proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -131,7 +131,9 @@ message WorkflowExecutionInfo {
     string new_execution_run_id = 61;
     temporal.server.api.clock.v1.ShardClock parent_clock = 62;
     // version of child execution initiated event in parent workflow
-    int64 parent_initiated_version = 63; 
+    int64 parent_initiated_version = 63;
+    int64 close_transfer_task_id = 64;
+    int64 close_visibility_task_id = 65;
```

Comment:
I am not sure if we want to make this more general, to something like lastTaskKey[category], which might be useful for detecting stuck workflow?

But I don't have strong opinion on this since I don't have a clear use case in mind right now and it might be a premature optimization.

Reply:
I considered this option too but having map for just 2 ints w/o proper understanding of future use seems to be too much.